AndroidProjects.yml

```diff
@@ -0,0 +1,9 @@
+name: Beginner-Level-AndroidStudio-Projects
+desc: this repository consists of some basic projects of android studio 
```

Comment:
@anshulkhattar Could this be improved to make it more clear?

Reply:
Made the desired changes, check it out.